README.md

```diff
@@ -1,4 +1,4 @@
-# formsy-react [![GitHub release](https://img.shields.io/github/release/christianalfoni/formsy-react.svg)](https://github.com/christianalfoni/formsy-react/releases) [![Build Status](https://travis-ci.org/christianalfoni/formsy-react.svg?branch=master)](https://travis-ci.org/christianalfoni/formsy-react)
+# formsy-react [![GitHub release](https://img.shields.io/github/release/formsy-react/formsy-react.svg)](https://github.com/formsy-react/formsy-react/releases) [![Build Status](https://travis-ci.org/formsy-react/formsy-react.svg?branch=master)](https://travis-ci.org/formsy-react/formsy-react)
```

Comment:
I was able to rename this org to 'formsy' a few minutes ago! 🎉 

A quick update to these docs and this will be good to go.

Reply:
Nice! Fixed.